agent_packager/tests/test_agent_packager.py

```diff
@@ -230,19 +230,44 @@ def test_create_agent_package(self):
             self.assertNotIn(excluded_module, pip_freeze_output)
         shutil.rmtree(TEST_VENV)
 
-    def test_dryrun(self):
+    def test_create_agent_package_in_existing_venv_force(self):
         cli_options = {
             '--config': CONFIG_FILE,
             '--force': True,
+            '--dryrun': False,
+            '--no-validation': False,
+            '--verbose': True
+        }
+        utils.make_virtualenv(TEST_VENV)
+        cli._run(cli_options)
+
```

Comment:
if a virtualenv already existed and we force an installation, we shouldn't delete the virtualenv at the end of the process.